src/util/server.ml

```diff
@@ -244,6 +244,20 @@ let () =
         Response.Error.(raise (of_exn exn))
   end);
 
+  register (module struct
+    let name = "read_config"
+    type params = { fname: string } [@@deriving of_yojson]
+    type response = unit [@@deriving to_yojson]
+    let process { fname } _ =
+      try
+        GobConfig.json_conf := Options.defaults;
+        Arg.current := 0;
```

Comment:
Better to move this into `parse_arguments` with a comment that it's necessary to reset in server mode.